app/connectors/EmailClient.scala

```diff
@@ -178,11 +163,28 @@ trait CSREmailClient extends OnlineTestEmailClient with AssessmentCentreEmailCli
 
 }
 
-trait OnlineTestEmailClient {
+sealed trait OnlineTestEmailClient {
+
+  self: EmailClient =>
+
   def sendOnlineTestInvitation(to: String, name: String, expireDateTime: DateTime)(implicit hc: HeaderCarrier): Future[Unit]
   def sendOnlineTestExpired(to: String, name: String)(implicit hc: HeaderCarrier): Future[Unit]
-  def sendTestExpiringReminder(to: String, name: String, timeLeft: Int,
-    timeUnit: TimeUnit, expiryDate: DateTime)(implicit hc: HeaderCarrier): Future[Unit]
+  def sendTestExpiringReminder(to: String, name: String, timeLeftInHours: Int,
+                               timeUnit: TimeUnit, expiryDate: DateTime)(implicit hc: HeaderCarrier): Future[Unit]
+
+  protected def sendExpiringReminder(template: String)(to: String, name: String, timeLeftInHours: Int,
```

Comment:
Why not include template in the rest of parameters? Is Currying helping in something here?